corehq/apps/linked_domain/models.py

```diff
@@ -103,6 +103,11 @@ def link_domains(cls, linked_domain, master_domain, remote_details=None):
             link = active_links_with_this_domain[0]
         else:
             # make a new link
+            if '/' in linked_domain and linked_domain[-1] != '/':
```

Comment:
can we say that there are remote details it would a remote app instead of checking if it has a slash? So this would be
`if remote_details and linked_domain[-1] != '/'`

and why not we just add one if its missing?

Reply:
Updated conditional in 03adfda1ae5

Not going to add the slash because altering the value of `linked_domain` will complicate this function - like, further up there's a test for `linked_domain` already existing, so if I update it I'd need to re-check that.